### PR TITLE
Delete old build files

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

File: src/gen.sh [hunk 1]
```diff
@@ -2,5 +2,9 @@
 #Concatenate files into one services file
 SRCDIR=$(dirname $0)
 cd "$SRCDIR"
+
+# Delete old build files
+rm -r ../build/
```
Comment (jlhughes) [inline src/gen.sh:7]: does 'make clean' not do this? or you want to run it each time?

Comment (hchoura) [reply src/gen.sh:7]: No. 
make clean do:
rm -f build/foam2.jar
rm -f build/foam2.classpath.txt
rm -rf .foam2-gensrcs

Comment (jlhughes) [reply src/gen.sh:7]: so why not add this to make clean?

Comment (jlhughes) [reply src/gen.sh:7]: with the clean change above, this is no longer needed correct?

**Comment by jlhughes** [comment]
Superceded by https://github.com/foam-framework/foam2/pull/2639

**Pull request closed without merging**